app/controllers/admin_controller.rb

```diff
@@ -371,6 +371,11 @@ def queue
     end
   end
 
+  def test_digest_email_spam
+    DigestSpamJob.perform_async(0)
```

Comment:
At what point are you calling/invoking this `test_digest_email_spam` ?

Reply:
This is just for testing, there is a button in /spam where I am making a post request. this will produce daily digest for moderators

Reply:
Gotcha, Thanks